backend/src/main/scala/com/softwaremill/bootzooka/infrastructure/DB.scala

```diff
@@ -19,16 +19,21 @@ import scala.util.NotGiven
 import scala.util.control.{NoStackTrace, NonFatal}
 
 class DB(dataSource: DataSource & Closeable) extends Logging with AutoCloseable:
+  private val transactor = Transactor(
+    dataSource = dataSource,
+    sqlLogger = SqlLogger.logSlowQueries(200.millis)
```

Comment:
this will use j.u.Logging? Or can I pass my own logger (`logger` from slf4j here) somehow?

Reply:
Magnum only uses the java.lang.System logger, a facade that should automatically work with whatever logging framework is used by the application (defaulting to ju logging)

https://stackoverflow.com/questions/59976828/difference-between-java-util-logging-logger-and-java-lang-system-logger

Reply:
I had to add one dependency to add a bridge to slf4j and now it works great :)

Reply:
Ah, I forgot that slf4j needed a bridge. Thanks for testing it :laughing: